Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+As a developer, understanding the fundamentals of the design tools used within your team is an essential skill. The notorious war between
+developers and designers is never-ending and design tools are the places where these two worlds collide. This raises the importance of using
+good design tools which provide a nice connection between the design and development process. Imagine if all the screen designs, components,
+different assets were stored on Google Drive and the review process and communication was handled through email/Slack. To get a single
+component, you'll have to go through several folders and then, you'll have to open your email/Slack to remind yourself about the discussions
+you had regarding the implementation. This isn't a big deal if you have to do it once in a day, however, this is a continuous and frequent
+process. Eventually, the project grows, new designs are requested, new folders and files are added and all of a sudden, handling designs
+like this sounds like a nightmare, right?
+
+![Designers vs Developers meme](/img/figma/figma_designers_vs_developers_meme.png "Designers vs Developers meme")
+
+[Figma](https://www.figma.com) is a cloud-based design tool which provides simple, effective and intuitive functionalities perfect for team
+collaboration. Similar to Google Docs, team members can easily communicate, collaborate and edit the same file in Figma in real time.
+
+## Benefits of using Figma
+
+- **Simple and intuitive**
+
+Most of us are already very familiar with using tools like Google Docs and Figma isn't much different from that. First-time Figma users can
+get quickly adjusted to using Figma and it won't take them too long to understand most of its functionalities.
+
+- **Easier team collaboration and communication**
+
+Communication and collaboration within Figma is extremely easy and convenient since Figma allows you to leave comments on the designs.
```

Comment:
Communication and collaboration within Figma **are** extremely easy

Reply:
I believe it can be both, but I'm not 100% sure 😅  

BTW @KCeh if you're sure about a correction, it would be cool if you could format your comment as a Github suggestion (cmd+g), so that the author can immediately commit it without having to manually do the changes and push 😄

Reply:
I used singular form because I pictured communication and collaboration as a whole, like some kind of a process, so even though it is a compound subject, I treated it as a singular idea. I don't really know how to explain it honestly hahaha 😅 (it's basically something like the dilemma mentioned in the last section "Compound Subjects: Singular vs Plural" of [this article](https://www.bkacontent.com/gs-clearing-common-issues-compound-subjects/)) I made the change here as well as in some other places where I had a similar approach for using singular instead of plural. 😄 
 
Thanks a lot for the comment, seems simple, but it left me wondering for a while hahaha 😅 I couldn't understand how I didn't notice the same mistake multiple times. 🤯 This PR turned into English 101 really quickly 😂